Allure.NUnit.AsyncTests/Allure.NUnit.AsyncTests.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
There is no need to create another solution - you can use Allure.NUnit.Examples as well

Reply:
Done, I've moved my tests to `Allure.NUnit.Examples . AllureAsyncLifeCycleTests`